website/cue/reference/components/sinks/elasticsearch.cue

```diff
@@ -73,307 +74,7 @@ components: sinks: elasticsearch: {
-							unit:    "seconds"
-						}
-					}
-				}
-			}
-		}
-		doc_type: {
-			common:      false
-			description: "The `doc_type` for your index data. This is only relevant for Elasticsearch <= 6.X. If you are using >= 7.0 you do not need to set this option since Elasticsearch has removed it."
-			required:    false
-			type: string: {
-				default: "_doc"
-			}
-		}
-		endpoint: {
-			common:      false
-			description: "The Elasticsearch endpoint to send logs to. This should be the full URL as shown in the example."
-			warnings: ["This option has been deprecated, the `endpoints` option should be used instead."]
-			required: false
-			type: string: {
-				default: null
-				examples: ["http://10.24.32.122:9000", "https://example.com", "https://user:password@example.com"]
-			}
-		}
-		endpoints: {
-			description: "The Elasticsearch endpoints to send logs to. Each endpoint should be the full URL as shown in the example."
-			required:    true
-			type: array: {
-				examples: [["http://10.24.32.122:9000"], ["https://example.com", "https://user:password@example.com"]]
-				items: type: string: {}
-			}
-		}
-		id_key: {
-			common:      false
-			description: "The name of the event key that should map to Elasticsearch's [`_id` field](\(urls.elasticsearch_id_field)). By default, Vector does not set the `_id` field, which allows Elasticsearch to set this automatically. You should think carefully about setting your own Elasticsearch IDs, since this can [hinder performance](\(urls.elasticsearch_id_performance))."
-			required:    false
-			type: string: {
-				default: null
-				examples: ["id", "_id"]
-			}
-		}
-		metrics: {
-			common:      false
-			description: "Options for metrics."
-			required:    false
-			type: object: {
-				examples: []
-				options: {
-					host_tag: {
-						common:      false
-						description: "Tag key that identifies the source host."
-						required:    false
-						type: string: {
-							default: "hostname"
-							examples: ["host", "hostname"]
-						}
-					}
-					timezone: configuration._timezone
-				}
-			}
-		}
-		mode: {
-			common:      true
-			description: "The type of index mechanism. If `data_stream` mode is enabled, the `bulk.action` is set to `create`."
-			required:    false
-			type: string: {
-				default: "bulk"
-				examples: ["bulk", "data_stream"]
-			}
-		}
-		pipeline: {
-			common:      true
-			description: "Name of the pipeline to apply."
-			required:    false
-			type: string: {
-				default: null
-				examples: ["pipeline-name"]
-			}
-		}
-		query: {
-			common:      false
-			description: "Custom parameters to Elasticsearch query string."
-			required:    false
-			type: object: {
-				examples: [{"X-Powered-By": "Vector"}]
-				options: {
-					"*": {
-						common:      false
-						description: "Any query key"
-						required:    false
-						type: string: {
-							default: null
-							examples: ["Vector"]
-							syntax: "literal"
-						}
-					}
-				}
```

Comment:
Note that so far I've been unable to get this same rendering in the base file for examples and syntax of the type.

Can't `&` it with the base one because it conflicts.

Reply:
Resolved per discussion with Toby. Essentially both the literal, and the examples can be ignored and don't need to be rectified.